modernz.lua

```diff
@@ -925,18 +925,17 @@ local function draw_slider_handle(element, elem_ass, override_alpha)
             ass_append_alpha(elem_ass, element.layout.alpha, override_alpha or 0)
             elem_ass:merge(element.static_ass)
         end
-    end
 
-    return xp, rh
+        return xp, rh
+    end
+    return xp, 0
 end
 
--- Draws slider seekranges according to user_opts 
-local function draw_slider_seek_ranges(element, elem_ass, xp, rh, override_alpha)
+-- Draws seekbar ranges according to user_opts 
+local function draw_seekbar_ranges(element, elem_ass, xp, rh, override_alpha)
+    local handle = ( xp and (mouse_hit_coords(element.hitbox.x1, element.hitbox.y1, element.hitbox.x2, element.hitbox.y2) or state.mouse_down_counter > 0) ) or ( xp and user_opts.handle_always_visible )
```

Comment:
Don't have the time to test right now but this should probably be simplified to:
```lua
local handle = xp and rh
```
Because `draw_seekbar_handle` returns `rh` as `0` when `should_display_handle == true`.  This should also work with the persistent seekbar as that passes `nil` for `rh`. Thus improving maintainability of multi state seekbars.